HomeworkTen/CalculatorMethods.java

```diff
@@ -0,0 +1,30 @@
+package HomeworkNine;
+
+public class CalculatorMethods {
+
+    public static int add (int number1, int number2) {
+        return number1 + number2;
+    }
+
+    public static int sub(int number1, int number2) {
+        return number1 - number2;
+    }
+
+    public static int mul (int number1, int number2) {
+        return number1 * number2;
+    }
+
+    public static double div (int number1, int number2) {
+        if (number2 != 0.0){
```

Comment:
you don't have to compare number2 with 0.0, because number2 has int type (just compare with 0)